consensus/quorum/quorum.go

```diff
@@ -187,14 +187,15 @@ func (s *cIdentities) IndexOf(pubKey bls.SerializedPublicKey) int {
 	return -1
 }
 
-func (s *cIdentities) NextAfter(pubKey *bls.PublicKeyWrapper) (bool, *bls.PublicKeyWrapper) {
+// NextAfter return the next pubkey, next can be negative number
+func (s *cIdentities) NextAfter(pubKey *bls.PublicKeyWrapper, next int) (bool, *bls.PublicKeyWrapper) {
```

Comment:
should rename this func now the logic is different. It's not "next" anymore. It's NthAfter

Reply:
sure.